src/components/modals/SharingModal.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Modal, Button, OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { FaShareAlt } from 'react-icons/fa';
+import { FaShareAltSquare } from 'react-icons/fa';
+
+const onCopyShareLink = (e) => {
+  navigator.clipboard.writeText(window.location.href);
+};
+
+const onShareLink = (e) => {
+  navigator.share({
+    url: window.location.href,
+    title: "LeapChat",
+    text: "Join me on LeapChat"
```

Comment:
Executive callout: if we enable sharing this way, this text might be the first thing new users potentially see when invited to the platform. Placeholder text; verbiage feedback humbly requested.

Reply:
Sounds good